changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.8.0 (Feature release)
+
+- Allow sharing of data between the server and client with `@financial-times/dotcom-ui-data-embed`.
```

Comment:
Super pedantic, but there's a convention of starting each changelog entry with the package name, e.g. `Data Embed plugin: Added a new package to provide a mechanism for passing data from server to client` and for better/worse, we don't tend to include much technical detail 🤷

Reply:
Will amend, I think "Allow sharing of data between the server and client" has just enough technical detail, do you agree?